vue-client/src/utils/record.js

```diff
@@ -165,39 +165,17 @@ export function getDigitalReproductionObject(original, resources) {
       digitalReproObject.mainEntity.instanceOf = original.mainEntity.instanceOf;
     }
   }
-
-  // Copy the PrimaryPublication if there is one
-  function getPrimaryPublication(mainEntity) {
-    if (mainEntity.hasOwnProperty('publication')) {
-      for (let i = 0; i < mainEntity.publication.length; i++) {
-        if (mainEntity.publication[i]['@type'] === 'PrimaryPublication') {
-          return mainEntity.publication[i];
-        }
-      }
-    }
-    return null;
-  }
-  const primaryPublication = getPrimaryPublication(original.mainEntity);
-  if (primaryPublication !== null) {
-    digitalReproObject.mainEntity.publication = [primaryPublication];
-  }
-
+  
   // Copy the other keys we want to copy
   const keysToCopy = [
     'mainEntity.hasTitle',
-    'mainEntity.responsibilityStatement',
-    'mainEntity.extent',
   ];
   for (let i = 0; i < keysToCopy.length; i++) {
     const originalValue = get(original, keysToCopy[i]);
     if (typeof originalValue !== 'undefined') {
       set(digitalReproObject, keysToCopy[i], originalValue);
     }
   }
-  // Add "indirectly identified by" with the "identified by" value from original
-  if (original.mainEntity.hasOwnProperty('identifiedBy')) {
-    digitalReproObject.mainEntity.indirectlyIdentifiedBy = original.mainEntity.identifiedBy;
```

Comment:
Are we OK with copying them all? (We may very well be; just double-checking.)

Reply:
`keysToCopy` only contains `'mainEntity.hasTitle'` so I should just remove the loop

Reply:
Ah, that's true. I was only thinking about `identifiedBy` though; i.e. copying all of those (may be more than one)...